__tests__/product/product.mutation.test.js

```diff
@@ -387,50 +372,44 @@ describe('Product mutations', () => {
     const productAfterUpdate = updateProduct.data.updateProduct;
     expect(productAfterUpdate).toBeDefined();
     expect(productAfterUpdate).toHaveProperty('name', [
-      { __typename: 'Language', value: 'Bad Baggy' },
-      { __typename: 'Language', value: 'Жахливий Рюкзачечок' },
+      { value: 'Bad Baggy' },
+      { value: 'Жахливий Рюкзачечок' },
     ]);
     expect(productAfterUpdate).toHaveProperty('description', [
-      { value: 'Baggy is so bad', __typename: 'Language' },
-      { value: 'Рюкзачечок - не добрий))', __typename: 'Language' },
+      { value: 'Baggy is so bad' },
```

Comment:
Please, don't hardcode values which you assert, instead use properties of the object that you have used for the creation of the product. It will reduce the required changes if some data of the product is going to be changed

Reply:
solved